WalletWasabi.Fluent/Views/AddWallet/WelcomePageView.axaml

```diff
@@ -26,39 +26,40 @@
 
     <DockPanel LastChildFill="True">
 
+      <!-- TODO: Uncomment in the case there is more than 1 welcome screen. -->
       <!-- Indicator & Buttons -->
-      <ListBox HorizontalAlignment="Center"
-               SelectedIndex="{Binding SelectedIndex}"
-               Background="Transparent"
-               IsHitTestVisible="False"
-               DockPanel.Dock="Bottom">
-        <ListBox.Styles>
-          <Style Selector="ListBoxItem:selected /template/ ContentPresenter#PART_ContentPresenter">
-            <Setter Property="Background" Value="Transparent" />
-          </Style>
-          <Style Selector="ListBoxItem:selected /template/ ContentPresenter">
-            <Setter Property="TextBlock.Foreground" Value="{DynamicResource SystemChromeAltLowColor}" />
-          </Style>
-          <Style Selector="ListBoxItem">
-            <Setter Property="Opacity" Value="0.3" />
-          </Style>
-          <Style Selector="ListBoxItem:selected">
-            <Setter Property="Opacity" Value="1" />
-          </Style>
-          <Style Selector="Ellipse">
-            <Setter Property="Fill" Value="{DynamicResource SystemChromeAltLowColor}" />
-            <Setter Property="Width" Value="9" />
-            <Setter Property="Height" Value="9" />
-          </Style>
-        </ListBox.Styles>
-        <ItemsControl.ItemsPanel>
-          <ItemsPanelTemplate>
-            <StackPanel Orientation="Horizontal" Spacing="5" />
-          </ItemsPanelTemplate>
-        </ItemsControl.ItemsPanel>
-        <Ellipse />
-        <Ellipse />
-      </ListBox>
+      <!-- <ListBox HorizontalAlignment="Center" -->
```

Comment:
It seems this can be removed.

Reply:
Since we have only 1 page now, the whole View could be refactored and the complete ViewModel too, as both designed for having multiple pages. 
Don't want to do that now as there are plans with the welcome screens. e.g: https://github.com/zkSNACKs/WalletWasabi/issues/13008

Reply:
Try to avoid letting garbage in the code. Nothing is lost, the version control is precisely for that.